packages/components/Tabs/src/TabsItem.settings.windows.ts

```diff
@@ -4,6 +4,8 @@ import type { IComposeSettings } from '@uifabricshared/foundation-compose';
 import type { TabsItemType } from './TabsItem.types';
 import { tabsItemName } from './TabsItem.types';
 
+export const tabsItemSelectActionLabel = 'Select a TabsItem';
```

Comment:
Was this an artifact of rebasing? Can you revert it?

Reply:
Had added this to fix some export issue, checked again, looks like it is no longer required. Removing it